data-access-analysis-tools/index-portal-intro.rst

```diff
@@ -119,9 +121,59 @@ Note that corresponding plot point for the selected row in the table is differen
+
+**3. Advanced ADQL**: When ADQL is selected as the query type, the interface in  **3.** changes to provide a free-form block into which ADQL queries can be entered directly. The query excuted in the :ref:`single-table-queries` example above can be expressed in ADQL as follows:
+
+.. code-block:: SQL
+
+   SELECT ra, dec, mag_g, mag_i
+   FROM dp01_dc2_catalogs.object
+   WHERE CONTAINS(
+   POINT('ICRS', ra, dec),
+   CIRCLE('ICRS', 61.863, -35.79, 0.05555555555555555))=1
+   AND (mag_g <24 AND mag_i <24)
+
+Type the above query into the ADQL Query block and click on the 'Search' button in the bottom left corner to execute. You should set the row limit to be a small number, such as 10,  when first testing queries. The search results will populate the same **Results View**, as shown above using the Single Table Query interface. A total of 205 records should be returned, which you can interact with in the sam manner as outlined in :ref:`single-table-queries`.
+
+**Joining with another tables**
+ It is often desireable to access data stored in more than just one table. We do this using a JOIN clause to combine rows from two or more tables. Here we will join the data in the object table with the data in the truth table to compare the results of the processing with the input truth information. The two tables are joined by matching the objectIds across two catalogs.
+
+.. code-block:: SQL
+
+   SELECT obj.ra, obj.dec, obj.mag_g, obj.mag_i, truth.ra, truth.dec, truth.mag_g, truth.mag_i
+   FROM dp01_dc2_catalogs.object as obj
+   JOIN dp01_dc2_catalogs.truth_match as truth
+   ON truth.match_objectId = obj.objectId
+   WHERE CONTAINS(
+   POINT('ICRS', ra, dec),
+   CIRCLE('ICRS', 61.863, -35.79, 0.05555555555555555))=1
+   AND (mag_g <24 AND mag_i <24)
+   AND truth.match_objectid != -1
+   AND truth.is_good_match = 'true'
+
+This query includes some additional filtering so that we In the truth_match table, '-1' indicates an unmatched object, so we will exclude them from the query. (TODO: How do 'is_good_match and match_object_id work together?)
+
+**Query the TAP service schema**
+Information about the LSST TAP schema can also be obtained via ADQL queries.  The following query gets the names of all the avaiabe DP0.1 tables.
+
+.. code-block:: SQL
+
+   SELECT * 
+   FROM tap_schema.tables 
+   WHERE tap_schema.tables.table_name like 'dp01%'
+
+To get the detailed list of columns available in the the Object table, their associated units and descriptions:
+
+.. code-block:: SQL
+
+   SELECT tap_schema.columns.column_name, tap_schema.columns.unit, tap_schema.columns.description 
+   FROM tap_schema.columns 
+   WHERE tap_schema.columns.table_name = 'dp01_dc2_catalogs.object'
+
```

Comment:
Would it make sense to include screen shots showing the results of these schema queries? The additional context might be helpful.

Reply:
yes !